.github/workflows/tests.yaml

```diff
@@ -24,7 +24,10 @@ jobs:
       - name: Add to PATH
         shell: bash
         run: |
-          echo "/home/runner/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
+          echo "~/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
```

Comment:
I've already merged in `getCustomErrorMessage()` so once #9 is merged this PR will look a lot smaller.